viewSQLposts.php

```diff
@@ -0,0 +1,37 @@
+<?php
+include ('include/SQLProject.php');
+?>
+
+
+<html>
+	<head>
+		<title> View Blog Posts </title>
+		<link rel='stylesheet' href='maindesign.css' />
+	</head>
+</html>
+
+<?php
+include('config/config.php');
+include('include/db_query.php');
```

Comment:
I think I sent you a link to my example code last week. I'd encourage you to look at how I included files. Over time you'll end up with more and more files in the include folder. It's easiest to just always have access to all of them, so I'd make a single file (I think mine was called `include_all.php` which includes all your other include files. Then on each of your main web pages like this one, you just include `include_all.php` and you have access to all the functions you've defined in all of your files.